pwa/src/components/organizationCard/OrganizationCard.module.css

```diff
@@ -0,0 +1,66 @@
+.container {
+  border: var(--web-app-component-card-border-color);
```

Comment:
Two notes:

1. We're passing a `..-border-color` token into a `border` property (I see that this actually hold an entire border)
2. We're passing a `-component-card` prop to a `organization-card` component here, maybe a more generic name (and the above note also in mind) could be: `--web-app-card-border`.